tests/unit/test_zos_files.py

```diff
@@ -102,3 +102,21 @@ def test_rename_dataset_member_raises_exception(self):
             Files(self.test_profile).rename_dataset_member("MY.DS.NAME", "MEMBER1", "MEMBER1N", "RANDOM")
 
         self.assertEqual(str(e_info.exception), "Invalid value. Valid options are SHRW or EXCLU.")
+
+    @mock.patch('requests.Session.send')
+    def test_rename_dataset_sends_specific_request(self, mock_send_request):
```

Comment:
This test still covers only one case. Please implement parametrized test that also covers values both expected and unexpected values of `enq`.

Ref: https://stackoverflow.com/questions/32899/how-do-you-generate-dynamic-parameterized-unit-tests-in-python